packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -199,12 +201,32 @@ function getWebExtensionPath(
 }
 
 function isSideloadingSupportedForDesktopHost(app: OfficeApp): boolean {
-  if (app === OfficeApp.Outlook || app === OfficeApp.Project || app === OfficeApp.OneNote) {
+  if (app === OfficeApp.Outlook && process.platform !== "win32" || app === OfficeApp.Project || app === OfficeApp.OneNote) {
```

Comment:
As we discussed, let's have a way to opt-in to the sideload to Outlook feature since it hasn't rolled out to production yet. For example, allow an environment variable to be set to enable the feature.

You might consider having the logic be for the true cases rather than the false cases.

Reply:
I will add a flag to opt out of Outlook sideload as we discussed